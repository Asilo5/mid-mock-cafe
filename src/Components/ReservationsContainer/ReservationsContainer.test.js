import React from 'react';
import { shallow } from 'enzyme';
import ReservationsContaner from './ReservationsContainer';

describe('ReservationsContaner', () => {
    it('should match wrapper with all information passing through correctly', () => {
        let mockReserv= [
            {
                id: 1,
                name: 'bob',
                date: '11/20',
                time: '11;30',
                number: 1
            }
        ]
        let wrapper = shallow(<ReservationsContaner totalReservations={mockReserv}
                                          removeReservations={jest.fn()} />);
     expect(wrapper).toMatchSnapshot();   
    })
})