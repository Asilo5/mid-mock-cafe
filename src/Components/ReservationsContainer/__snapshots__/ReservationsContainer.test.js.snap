// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReservationsContaner should match wrapper with all information passing through correctly 1`] = `
<section
  className="reservations-container"
>
  <Reservation
    date="11/20"
    id={1}
    key="1"
    name="bob"
    number={1}
    removeReservations={[MockFunction]}
    time="11;30"
  />
</section>
`;
